recipes/mpl_sample_data/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "2.0.2" %}
+
+package:
+  name: mpl_sample_data
+  version: {{ version }}
+
+source:
+  fn: matplotlib-{{ version }}.tar.gz
+  url: https://github.com/matplotlib/matplotlib/archive/v{{ version }}.tar.gz
+  sha256: aebed23921562792b68b8ca355de5abc176af4424f1987e2fa95f65e5c5e7e89
+
+build:
+  number: 0
+
+requirements:
+  run:
+    - matplotlib
```

Comment:
If we go the `basemap` route the data will depend on `matplotlib` and not the other way around. But if you want the data the always be present we need to change this.